gradle.properties

```diff
@@ -2,6 +2,16 @@ projectVersion=4.0.0-SNAPSHOT
 
 # This prevents the Grails Gradle Plugin from unnecessarily excluding slf4j-simple in the generated POMs
 # https://github.com/grails/grails-gradle-plugin/issues/222
+gradleToolingApiVersion=8.10.2
+grailsGradlePluginVersion=7.0.0-SNAPSHOT
+grailsVersion=7.0.0-SNAPSHOT
+groovyVersion=4.0.24
```

Comment:
To ensure forward compatibility, you can drop springVersion, spocVersion, groovyVersion.  They should be in the bom.

Reply:
Fixed